Makefile

```diff
@@ -29,44 +29,25 @@ include .make/go/main.make
 include .make/protos/main.make
 include .make/js/main.make
 include .make/dev.make
-include .make/ci.make
 include .make/styl/main.make
 include .make/snap/main.make
 include .make/sdk/main.make
 
 messages:
 	@$(GO) run ./cmd/internal/generate_i18n.go
 
-dev-deps: go.dev-deps js.dev-deps
+dev-deps: go.deps js.dev-deps
 
 deps: go.deps js.deps sdk.deps
 
 test: go.test js.test sdk.test
 
 quality: go.quality js.quality styl.quality snap.quality
 
-clean: go.clean js.clean
```

Comment:
Why remove `make clean`?

Reply:
It's still here
https://github.com/TheThingsNetwork/lorawan-stack/pull/109/files/dd77b649359aa917a56980930e0a6056355e8381#diff-b67911656ef5d18c4ae36cb6741b7965R50